en/quiz1/question_1.md

```diff
@@ -9,74 +9,49 @@ A question may optionally have a 'frontmatter' section with the `legend` paramet
 --- question ---
 
 ---
-legend: Question 1 of 2
+legend: Question 1 of 3
 ---
 
-The bug sprite starts facing upwards.
+A dog sprite in Scratch has the following code:
 
-![Bug facing upwards](./img/q3-1.png)
+![A dog](images/quiz/q1-1.png) ![Selection of four scratch blocks](images/quiz/q1-2.png)
```

Comment:
needs to be '../images/quiz/q1-2.png'? etc. for other images?  If I view this file in the PR images are broken

Reply:
Yeah, but I'm not sure if the processing will magically fix? @sHtev ?

Reply:
If you preview the other projects like Rock Band, the images in GH show up.  I don't think we do anything in Kramdown that's custom for images?